src/pages/settings/Profile/ProfilePage.js

```diff
@@ -148,8 +148,7 @@ class ProfilePage extends Component {
                         onClose={PersonalDetails.clearAvatarErrors}
                     >
                         <AvatarWithImagePicker
-                            isUsingDefaultAvatar={lodashGet(currentUserDetails, 'avatar', '').includes('/images/avatars/avatar')}
```

Comment:
I think we still need this prop which disables the "Remove" option when the avatar is default - right?

Reply:
yes! nice catch :)